app.js

```diff
@@ -134,6 +156,7 @@ function getToolTooltip(id) {
 
 function createToolListMarkup(parentEl) {
   var keys = Object.keys(COMPONENT_MAPPING);
+  var categoryInUrl = getParameterByName("category");
```

Comment:
I would rename `category` to `tool`. We don't use the word category anywhere in the code or the UI, and tool seems to me like a better fit. Also easier to type.